cmd/interceptors/main.go

```diff
@@ -72,8 +89,62 @@ func main() {
 	mux.Handle("/", service)
 	mux.HandleFunc("/ready", handler)
 
+	name := os.Getenv("SVC_NAME")
```

Comment:
IMO we should give this a more explicit name than SVC name e.g. INTERCEPTOR_TLS_SECRET or something like that